src/nemos/glm.py

```diff
@@ -407,6 +441,19 @@ def score(
             If X structure doesn't match the params, and if X and y have different
             number of samples.
 
+        Examples
+        --------
+        >>> # example input
+        >>> import numpy as np
+        >>> X, y = np.random.normal(size=(10, 2)), np.random.uniform(size=10)
```

Comment:
```suggestion
        >>> X, y = np.random.normal(size=(10, 2)), np.random.poisson(size=10)
```

The default PoissonObservations model discrete variables, if you pass a continuous variable, the function you are maximizing is not a likelihood anymore.